### ISSUE TITLE
Service Intent must be explicit in Android 5.0

### ISSUE THREAD

**Issue by woodpd** [phonegap-build/GAPlugin#86]
When we build our app with the GA Plugin and Android target SDK version = 21 (Lollipop) it crashes on Android 5.0 every time and the stack trace reports that the GA service intent must be explicit. This is explained in the SO post linked below and is related to changes in Android that need to be catered to - please address this as we're having to run as Target SDK version 19 at the moment

http://stackoverflow.com/questions/26530565/android-5-0-l-service-intent-must-be-explicit

**Comment by iwantwin** [comment]
Yes, the problem for this specific plugin seem to be solved when the google analytics plugin library will be updated from version v2 to v3 - Changeguide can be found here: https://developers.google.com/analytics/devguides/collection/android/v3/migration

**Comment by woodpd** [comment]
Hey @shazron noticed you mentioned in #64 that you'd be looking at upgrading Android after you finished the iOS upgrade. Is this still on your roadmap?

**Comment by woodpd** [comment]
Also, looks like Google now have a v4 out - https://developers.google.com/analytics/devguides/collection/android/v4/

**Comment by rlisperguier** [comment]
i'm having the same problem, any estimated date for the sdk update?

Thanks

**Comment by levi-cit** [comment]
Any update on this issue?

**Comment by umer-ashraf** [comment]
Guys any update on this?

**Comment by Carlos487** [comment]
It would be nice if it is corrected at a plugin level, al least at the moment the workaround to use targetsdk 19 has worked, but it was needed to change in androidmanifest.xml and in project.properties

**Comment by 123AndGo** [comment]
Having the same problem, any update? Thanks!

**Comment by Carlos487** [comment]
If you need to use other plugins that rely on android target 22 my workarround doesn't work anymore, looking for an alternate plugin. 

Like [cmackay](https://github.com/cmackay/google-analytics-plugin) and  [danwilson](https://github.com/danwilson/google-analytics-plugin). 

It'ss sad that people from phonegap haven't even acknowledge or comment about this issue.
